app/repository/pg/user.go

```diff
@@ -64,3 +64,15 @@ func (r Repo) FindUser(id string) (*entities.User, error) {
 
 	return &user, nil
 }
+
+func (r Repo) Delete(user entities.User) error {
+	const sqlStatement = `UPDATE "user" SET deleted_at = NOW() WHERE id=($1) AND deleted_at IS NULL RETURNING id`
+
+	if err := r.DB.QueryRow(sqlStatement, user.ID).Scan(&user.ID); err != nil {
```

Comment:
What if you receive any other error? Don't you want to return them back?

Reply:
Thank you! We fixed it